static/app/views/alerts/rules/issue/ticketRuleModal.tsx

```diff
@@ -20,13 +20,13 @@ type Props = {
   index: number;
   // The AlertRuleAction from DB.
   instance: IssueAlertRuleAction;
+  link: string | null;
```

Comment:
`link` will always be set for this action type. We allow `null` b/c the GitHub + GitHub Enterprise integrations don't have issue sync yet, so link is set to `None` for those classes